src/presenters/admin/indexation-list-item-presenter.php

```diff
@@ -59,15 +72,27 @@ public function present() {
 			$output .= '<span class="wpseo-checkmark-ok-icon"></span>' . \esc_html__( 'Great, your site has been optimized!', 'wordpress-seo' );
 		}
 		else {
+
+			$should_index = $this->indexable_helper->should_index_indexables();
+			$disabled     = ( $should_index ) ? '' : 'disabled';
+
 			$output .= \sprintf(
 				'<span id="yoast-indexation">' .
-					'<button type="button" class="button yoast-open-indexation" data-title="%1$s" data-settings="yoastIndexationData">' .
-						'%2$s' .
-					'</button>' .
+				'<button type="button" class="button yoast-open-indexation" data-title="%1$s" data-settings="yoastIndexationData" ' .
```

Comment:
I would add the space before the `'disabled'` in variable `$disabled`
```suggestion
				'<button type="button" class="button yoast-open-indexation" data-title="%1$s" data-settings="yoastIndexationData"' .
```

Reply:
this will all be removed as part of P2-182 so we decided not to stop spending time on it.